backend/item/repo.go

```diff
@@ -0,0 +1,66 @@
+package item
+
+import (
+	"github.com/SAKA-club/todo/backend/gen/models"
+	"github.com/jmoiron/sqlx"
+	"github.com/rs/zerolog/log"
+	"time"
+)
+
+type repo struct {
+	db *sqlx.DB
+}
+
+func NewRepo(db *sqlx.DB) *repo {
+	return &repo{db: db}
+
+}
+
+func (r repo) GetAll() ([]*models.Item, error) {
+	var items []*models.Item
+	if err := r.db.Select(&items, `SELECT id, title, priority, schedule_time, complete_time FROM item`); err != nil {
+		log.Err(err).Msg("could not get items from db")
+		return []*models.Item{}, err
+	}
+	return items, nil
+}
+
+func (r repo) Get(ID int64) (*models.Item, error) {
+	var item models.Item
+	if err := r.db.Get(&item, "SELECT  id, title, priority, schedule_time, complete_time FROM item WHERE id= $1", ID); err != nil {
+		log.Err(err).Msg("could not find by id")
```

Comment:
let's be specific about what could not be found. This will be helpful later on when when we need to debug our code.